lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorSettings.cs

```diff
@@ -174,12 +187,46 @@ public IRemoteManager<REEFMessage> RemoteManager
             }
         }
 
-        public IInjector Injector
+        public IInjector ClrDriverConfigInjector
         {
             get
             {
                 return _injector;
             }
         }
+
+        private Optional<TaskConfiguration> GetTaskConfiguration()
```

Comment:
Yes, I meant property.

Can't we have two constructors: One which takes the parameter and the other one which doesn't? Tang should call the most specific one possible, given the configuration.